content/en/blog/_posts/2016-08-00-Kubernetes-Namespaces-Use-Cases-Insights.md

```diff
@@ -125,7 +125,7 @@ You may wish to, but you cannot create a hierarchy of namespaces. Namespaces can
 
 
 
-Namespaces are easy to create and use but it’s also easy to deploy code inadvertently into the wrong namespace. Good DevOps hygiene suggests documenting and automating processes where possible and this will help. The other way to avoid using the wrong namespace is to set a [kubectl context](/docs/user-guide/kubectl/kubectl_config_set-context/).&nbsp;
+Namespaces are easy to create and use but it’s also easy to deploy code inadvertently into the wrong namespace. Good DevOps hygiene suggests documenting and automating processes where possible and this will help. The other way to avoid using the wrong namespace is to set a [kubectl context](/docs/user-guide/kubectl/kubectl_-em-set-context-em-).&nbsp;
```

Comment:
https://deploy-preview-28919--kubernetes-io-main-staging.netlify.app/docs/user-guide/kubectl/kubectl_-em-set-context-em-
eventually goes to 
https://deploy-preview-28919--kubernetes-io-main-staging.netlify.app/docs/reference/generated/kubectl/kubectl-commands#-em-set-context-em- thanks to the redirection rule,
so your change works of course,
but I suggest to update to the eventual target URL.
```suggestion
Namespaces are easy to create and use but it’s also easy to deploy code inadvertently into the wrong namespace. Good DevOps hygiene suggests documenting and automating processes where possible and this will help. The other way to avoid using the wrong namespace is to set a [kubectl context](/docs/reference/generated/kubectl/kubectl-commands#-em-set-context-em-).&nbsp;
```